runtime/appruntime/exported/config/config.go

```diff
@@ -44,13 +44,32 @@ type Runtime struct {
 	RedisServers    []*RedisServer          `json:"redis_servers,omitempty"`
 	RedisDatabases  []*RedisDatabase        `json:"redis_databases,omitempty"`
 	Metrics         *Metrics                `json:"metrics,omitempty"`
+	Services        []*Service              `json:"services,omitempty"`
+	Gateways        []*Gateway              `json:"gateways,omitempty"`
```

Comment:
It's not clear exactly what it means to have an entry here. Is it saying all gateways listed should be served by the running container?

Reply:
Added a comment